src/modules/45-dashboards/pages/home/CreateDashboardForm.tsx

```diff
@@ -8,17 +8,18 @@
 import React, { useState } from 'react'
 import { useParams, useHistory } from 'react-router-dom'
 import { ModalErrorHandlerBinding, Layout, Container, useToaster } from '@harness/uicore'
-import { CreateDashboardResponse, useCreateDashboard } from 'services/custom-dashboards'
+import { CreateDashboardResponse, FolderModel, useCreateDashboard } from 'services/custom-dashboards'
 import { useStrings } from 'framework/strings'
 import routes from '@common/RouteDefinitions'
 import type { DashboardPathProps } from '@common/interfaces/RouteInterfaces'
 import DashboardForm, { DashboardFormRequestProps } from './DashboardForm'
 
 interface CreateDashboardFormProps {
+  editableFolders: FolderModel[]
   hideModal: () => void
 }
 
-const CreateDashboardForm: React.FC<CreateDashboardFormProps> = ({ hideModal }) => {
+const CreateDashboardForm: React.FC<CreateDashboardFormProps> = ({ editableFolders = [], hideModal }) => {
```

Comment:
Looks like `editableFolders` isn't optional
```suggestion
const CreateDashboardForm: React.FC<CreateDashboardFormProps> = ({ editableFolders, hideModal }) => {
```